ocs_ci/resiliency/resiliency_workload_factory.py

```
@@ -94,22 +94,37 @@ def setup_workloads(self):
+                max_concurrent_operations=workload_config.get_background_operations_max_concurrent(),
             )
+            self.background_cluster_ops.start()
 
-            log.info("Background cluster operations started successfully")
+            ops_count = len(enabled_operations) if enabled_operations else "all"
+            log.info(
+                f"Background cluster operations started with {ops_count} "
+                f"operation types (interval: "
+                f"{workload_config.get_background_operations_interval()}s, "
+                f"max concurrent: "
+                f"{workload_config.get_background_operations_max_concurrent()})"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)